dali/operators/image/mask/grid_mask.cc

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/core/static_switch.h"
+#include "dali/operators/image/mask/grid_mask.h"
+#include "dali/kernels/mask/grid_mask_cpu.h"
+#include "dali/core/format.h"
+
+#define TYPES (uint8_t, int16_t, int32_t, float)
+
+namespace dali {
+
+DALI_SCHEMA(GridMask)
+    .DocStr(R"(Performs the gridmask augumentation (https://arxiv.org/abs/2001.04086).
+Zeroes out pixels of an image in a grid-like fashion. The grid
```

Comment:
```suggestion
    .DocStr(R"(Performs the gridmask augumentation (https://arxiv.org/abs/2001.04086).
    
Zeroes out pixels of an image in a grid-like fashion. The grid
```
According to how the docs should look like the second sentence should be in a new line compared to the first one. Blank line does that for rst.